src/main/scripts/generate_chart_repo.sh

```diff
@@ -51,16 +50,15 @@ echo "Regenerating chart repo index.yaml"
 # $PUBLISH_DIR for committing to git.
 docker run \
   --user "$(id -u):$(id -g)" \
-  -v "$(pwd)/$PUBLISH_DIR:/index" \
```

Comment:
We need the entire repo since 1.5.0 helm chart releaser expects a git repository.